pages/vi/vi-faq.md

```diff
@@ -3,101 +3,101 @@
 ## General Internship Questions
 
 - **Q1:** [What will I be working on during this internship?](#Q1:_What_will_I_be_working_on_during_this_internship?)
-- **Q2:** [What kind of software will we be using for this internship?](#Q2:_What_kind_of_software_will_we_be_using_for_this_internship?)
+- **Q2:** [What kind of software will I be using for this internship?](#Q2:_What_kind_of_software_will_I_be_using_for_this_internship?)
 - **Q3:** [Where can I find more information about this internship?](#Q3:_Where_can_I_find_more_information_about_this_internship?)
-- **Q4:** [What are the hours of the internship?](#Q4:_What_are_the_hours_of_the_internship?)
+- **Q4:** [What is the time commitment for this internship?](#Q4:_What_is_the_time_commitment_for_this_internship?)
```

Comment:
You have changed the meaning of question 4. It should stay as it was originally.